### DISCUSSION TITLE
How can I run react-figma on web ?

### DISCUSSION THREAD

**Discussion by zohra-alt** [react-figma/react-figma#439] [Q&A]
I want to run the code from React-Figma on the web as a react application, not on Figma desktop. How I can do that?

**Comment by ilyalesik** [comment]
Hey! I propose to use webpack aliasing:

```json
{
  "resolve": {
    "alias": {
      "react-figma$": "react-native-web",
    }
  }
}
```
please, check the [demo](https://github.com/react-figma/PrimerDemo)
